frontend-react/src/components/Admin/MessageTesting/MessageTestingForm.tsx

```diff
@@ -51,7 +51,7 @@ const MessageTestingForm = ({
                                     title={item.fileName}
                                     body={item.reportBody}
                                     handleSelect={() => handleSelect(item)}
-                                    selectedOption={selectedOption?.reportBody ? selectedOption.reportBody : null}
+                                    selectedOption={selectedOption ? selectedOption : null}
```

Comment:
This can be shortened
`selectedOption={selectedOption || null}`

Reply:
good point, TS will complain and make me do: `selectedOption={selectedOption ?? null}`